README.md

```diff
@@ -157,3 +157,8 @@ rMAPI will set the exit code to `0` if the command succeedes, or `1` if it fails
 - `RMAPI_CONFIG`: filepath used to store authentication tokens. When not set, rmapi uses the file `.rmapi` in the home directory of the current user.
 - `RMAPI_TRACE=1`: enable trace logging.
 - `RMAPI_USE_HIDDEN_FILES=1`: use and traverse hidden files/directories (they are ignored by default).
+
+# Development environment
+
```

Comment:
When I read this, it seems to imply that it's mandatory to use Nix to create your development environment and that's not the case.

How about we add a subsection here:

```suggestion
## Using Nix
```